### PR TITLE
CERNBulletin: changed menu URLs after 2016/41

### PR DESCRIPTION
**Pull request by jbenito3**
This PR addresses the following request:

> Could you also make some changes to the menu URLs?
> Specifically, the menu URLs for both CERN Bulletin and Echo Bulletin must change as follows from 16/10:
> 
> For English:
> News Articles - http://home.cern/cern-people
> Official News - http://home.cern/cern-people/official-communications
> Learning - Remove this one
> Announcements - http://home.cern/cern-people/announcements
> Events - http://home.cern/cern-people/events
> 
> For French:
> Actualités - http://home.cern/fr/cern-people
> Communications officielles - http://home.cern/fr/cern-people/official-communications
> Formations - Remove this one
> Annonces - http://home.cern/fr/cern-people/announcements
> Événements  - http://home.cern/fr/cern-people/events

### CODE REVIEW

File: modules/webjournal/lib/webjournal_utils.py [hunk 1]
```diff
@@ -1476,13 +1476,35 @@ def make_journal_url(current_uri, custom_parameters=None):
...
     uri = CFG_SITE_URL + '/journal/'
+
+    new_bulletin_urls = {
+        'News Articles': 'http://home.cern/%scern-people',
+        'Official News': 'http://home.cern/%scern-people/official-communications',
+        'Announcements': 'http://home.cern/%scern-people/announcements',
+        'Events': 'http://home.cern/%scern-people/events'
+    }
+
     if default_params['journal_name']:
```
Comment (ludmilamarian) [inline modules/webjournal/lib/webjournal_utils.py:1479]: Can you please use `https`?

**Comment by jbenito3** [comment]
@ludmilamarian changed. I have also refactored the code to ensure that "Learning" is not removed for past bulletin editions.

**Comment by jbenito3** [comment]
@CERNDocumentServer/cds-members I have deployed this PR in production. I would recommend to integrate it as soon as 'a maintainer' finds 5 mins :)

**Comment by jbenito3** [comment]
Oh, I see...labels :) Thanks @drjova

**Comment by ludmilamarian** [comment]
Thanks, I see it working well in prod. Can you please amend the commit message to follow the invenio rules? http://invenio.readthedocs.io/en/latest/technology/git.html#r2-remarks-on-commit-log-messages

**Comment by jbenito3** [comment]
@ludmilamarian could you be a bit more precise on which way I am not following the rules? 😸

**Comment by ludmilamarian** [comment]
@jbenito3 Sorry I did not add more details, it's just minor details, as component name and the tens of verbs. I would rephrase it as:

```
WebJournal: CERN navigation menu update 

* Updates the navigation menu URLs for all issues
  of CERNBulletin after 2016/41. 
```

**Comment by jbenito3** [comment]
@ludmilamarian ok, I see I messed up with the title mainly, the body (description) seems alright to me besides the lack of empty line between items...and I thought I knew the rules...

```
WebJournal: CERN navigation menu update

* Updates the navigation menu URLs for all issues after 2016/41
  following the migration of CERN Bulletin to the Drupal site.

* Fixes bug while importing webjournal utils from legacy.
```

Done and pushed...

**Comment by ludmilamarian** [comment]
Great, thanks! I am going to merge it now. One thing that is worth remembering for the future, is the update of the copyright years.
